From 955448e27f4cd3c6837fac2c599ed85395b2a095 Mon Sep 17 00:00:00 2001
From: Thiago Macieira <thiago.macieira@intel.com>
Date: Sun, 5 Aug 2018 09:37:13 -0700
Subject: [PATCH 1/2] Add more autoconf-compatibility options

---
 configure.py | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/configure.py b/configure.py
index c6d5d23..7d76dbb 100755
--- a/configure.py
+++ b/configure.py
@@ -646,7 +646,11 @@ def process_command_line(args):
         'psdir',
         'sbindir',
         'sharedstatedir',
-        'sysconfdir'
+        'sysconfdir',
+        'host',
+        'target',
+        'build',
+        'program-prefix'
         ]
 
     for opt in compat_with_autoconf_options:
-- 
2.43.0

