From 99eef3ee04f16d93e2487220b75b909f907336a2 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Wed, 6 Dec 2023 12:19:13 -0800
Subject: [PATCH 2/2] Enable genentropy(3) on Linux

---
 src/build-data/os/linux.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/build-data/os/linux.txt b/src/build-data/os/linux.txt
index 9659896..3c9fcc6 100644
--- a/src/build-data/os/linux.txt
+++ b/src/build-data/os/linux.txt
@@ -6,6 +6,7 @@ posix1
 posix_mlock
 
 dev_random
+getentropy
 clock_gettime
 getauxval
 
-- 
2.43.0

